FirebaseAuthUI.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = 'FirebaseAuthUI'
-  s.version      = '14.2.4'
```

Comment:
Can you update the FirebaseAuth dependency in this file too? (Line 21)

Reply:
Good catch!